test/e2e/conformance_test.go

```diff
@@ -97,21 +97,22 @@ var _ = Describe("Conformance Tests", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(os.Setenv("CI_VERSION", kubernetesVersion)).To(Succeed())
 			Expect(os.Setenv("CLOUD_PROVIDER_AZURE_LABEL", "azure-ci")).To(Succeed())
+		}
 
+		if flavor == "" {
```

Comment:
are these changes related to the new templates?

Reply:
Yes, without this then any run passing the `kubetest.use-ci-artifacts` or `kubetest.use-pr-artifacts` flags to the e2e binary will only use the vanilla ci-version/custom-builds template.